Add documentation dev new field with language (#2953)
GeotrekCE/Geotrek-admin: docs/development.rst

If ``MyModel`` is served by APIv2, make sure to add a serializer for the new field in ``geotrek/api/v2/serializers.py`` and if you wish to filter on this field, create a new filter and add it to the right ``ViewSet`` under ``geotrek/api/v2/views``, using attribute ``filter_backends``.

> When updating a field ``my_field`` to a model ``MyModel`` for ``new_field``, if this field is translated (check ``geotrek/{app}/translation.py``)

Reviewer: ```suggestion
When updating a field ``my_field`` in a model ``MyModel`` for ``new_field``, check if this field is translated in ``geotrek/{app}/translation.py``.
```

> You need to add a migration just after the migration generated by django.

Reviewer: ```suggestion
If so, you need to add a migration just after the migration generated by django.
```

> It should rename the old fields generated by modeltranslation ``my_field_en`` by ``new_field_en``

Reviewer: ```suggestion
This migration should rename the old fields generated by modeltranslation ``my_field_en`` by ``new_field_en``
```
